service/eventlog-server/README.md

```diff
@@ -0,0 +1,121 @@
+# Service: CCNP Eventlog Server
+
+To further verify the integrity and authenticity of the measurements in the confidential cloud native environment and its underlying platform, event logs are absolutely needed.
+By reviewing the event logs, user can identify any errors or issues that may be preventing the confidential environment from functioning correctly.
+
+This service (which is actually a GRPC server) will provide support to fetch the event logs for confidential cloud native environments, including TDX RTMR event logs and TPM event logs.
+
+
+
+## Introduction
+
+This service provides functionality to fetch event logs for confidential cloud native environments, both platform level(`PAAS` option) and container level (`SAAS` option). Using this service, user can fetch event logs for both TDX RTMR (`TDX_EVENTLOG` option) and TPM (`TPM_EVENTLOG` option).
+Here shows the proto buf for the service:
+
+```
+enum CATEGORY {
+    TDX_EVENTLOG = 0;
+    TPM_EVENTLOG = 1;
+}
+
+enum LEVEL {
+    PAAS = 0;
+    SAAS = 1;
+}
+
+message GetEventlogRequest {
+    LEVEL eventlog_level = 1;
+    CATEGORY eventlog_category = 2;
+    int32 start_position = 3;
+    int32 count = 4;
+}
+
+service Eventlog {
+    rpc GetEventlog (GetEventlogRequest) returns (GetEventlogReply) {}
+}
+```
+
+To select different level and category of event log, user shall send out request with different settings.
+The service also supports fetching number of event logs start from a certain one. The `start_position` option and `count` option are provided for the usage.
+User can find sample request in the Testing section.
+
+The collected event logs are stored under the folder `/run/ccnp-eventlog` by default. User can change the directory by changing the `eventlogDir` variable within the helm chart values.yaml file.
+
+
+
+## Installation
+
+The Eventlog server service can be deployed as either DaemonSet or sidecar in different user scenarios.
+
+
+### Prerequisite
+
+To simplify the deployment process, we use Helm as the basic tool for deployment. Please install Helm following the [Helm official guide](https://helm.sh/docs/intro/install/).
+
+### Build docker image
+
+The dockerfile for the service can be found under `container/eventlog-server` directory. Use the following command to build the image:
+
+```
+cd ../..
+docker build -t ccnp_eventlog_server:0.1 -f container/eventlog-server/Dockerfile .
+```
+> Note: if you are using containerd as the default runtime for kubernetes. Please remember to use the following command to import the image into containerd:
```

Comment:
if you are using containerd as the default runtime for kubernetes, please don't forget to use the following commands to import the image into containerd first: